EIPS/eip-3005.md

```diff
@@ -0,0 +1,340 @@
+---
+eip: 3005
+title: Batched meta transactions
+author: Matt (@defifuture)
+discussions-to: https://ethereum-magicians.org/t/eip-3005-the-economic-viability-of-batched-meta-transactions/4673 
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-09-25
+---
+
+## Simple Summary
+
+Defines an extension function for ERC-20 (and other fungible token standards), which allows receiving and processing a batch of meta transactions.
+
+![](../assets/eip-3005/meta-txs-directly-to-token-smart-contract.png)
+
```

Comment:
Motivation is the best place I can think of for this image.  It isn't necessary for the specification, which means it probably shouldn't be part of summary/abstract/specification, and rationale makes even less sense than Motivation.  😄